### ISSUE TITLE
Add default columns

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#541]
Some columns are more important than others. Therefore there should be some way of prioritising which columns are shown by default (assuming they exist in the data)

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
